Tests/test_testseq.py

```diff
@@ -0,0 +1,109 @@
+# Copyright 2017 by Adil Iqbal.  All rights reserved.
+# This code is part of the Biopython distribution and governed by its
+# license.  Please see the LICENSE file that should have been included
+# as part of this package.
+
+import os
+import unittest
+
+from Bio.Alphabet import IUPAC, NucleotideAlphabet
+from Bio.SeqUtils import GC
+
+
+def manually_import(name):
+    """Find, import, and return a module in Python 2.7."""
+    name = name.split(".")
+    name[-1] += ".py"
+    module_path = os.getcwd()
+    module_path = os.path.split(module_path)
+    if module_path[1] == "Tests":
+        module_path = os.path.join(module_path[0], name.pop(0))
+    else:
+        raise ImportError("Must call 'manually_import' from inside 'Tests' folder.")
+    for i, step in enumerate(name):
+        module_path = os.path.join(module_path, step)
+    name = name[-1][:-3]
+    import imp
+    return imp.load_source(name, module_path)
```

Comment:
If we do end up with more code in ``Scripts/`` being tested like this, then that seems like a good idea. Right now having the code here seems fine to me.